scripts/package-build

```diff
@@ -123,6 +123,38 @@ elif [[ $(type -t build) == "function" ]]; then
     error "A 'build()' function was defined but the 'image' variable is missing"
 fi
 
+function convert-draft-apps() {
```

Comment:
@matteodelabre Would it make more sense to move this to package-lib?

Reply:
I think I've answered this question as "Yes"

Reply:
Sure, we ideally want to keep the `package-build` script as lightweight as possible.

Reply:
I think I'll remove this and users can just use the less ideal "Import draft apps" menu option for everything other than KOReader.